### DISCUSSION TITLE
Web visualizer Piano Visualizer

### DISCUSSION THREAD

**Discussion by Grinchou** [onlaj/Piano-LED-Visualizer#255] [Q&A]
Hi everybody.

New here and also complete newbie in Raspi stuff.
I start a project with the piano visualizer.
I intend to use the same process as mentioned in this topic but without the mini screen.

Means that I would need the web  visualizer. Not understood how to get it.
I installed the program on the sd card then inserted to the Pi WH

The PI wh is now connected to my Mac.

Shall I need a specific external screen and keyboard to have an access to the web visualizer?

thanks

**Comment by onlaj** [comment]
Hey, sorry for late response.
Connect RPi to your network, [here](https://www.terminalbytes.com/raspberry-pi-without-monitor-keyboard/) is instruction, start from step 3.
Then open browser on any device connected to the same network and type your RPi's local IP address (for example 192.168.1.10).

**Reply by Grinchou** [reply]
tanks! will try it! Just also purchased the small LCD screen  Outstanding and immediate.
